CONTRIBUTING.md

```diff
@@ -1,70 +1,98 @@
+* 💬 **Suggestions via issues**
+* ✍️ *Practice reflections and insights* shared externally (e.g., Medium)
+* 🧩 Documentation, design, or structural improvements
 
-- Complete at least **Gray Belt level** in the [Mindset Dojo Program](https://dojo.center/program)
-- Understand our rhythm, principles, and tone of engagement
+**Pull requests (PRs)** are open to all, but deeper integration happens through practice. Before taking on core roles or becoming a regular contributor, we ask that you first:
+
+* Complete at least the ***Gray Belt*** level in the [https://dojo.center/program](https://dojo.center/program)
+* Understand our **rhythm**, **principles**, and **tone of engagement**
 
 This helps ensure alignment between our inner work and our outward contributions.
 
 ---
 
-## 💡 Our Contribution Principles
+## 💡 Principles of Practice
+
+In this dojo, our work is not just technical — it is a *shared practice*. The following principles guide how we contribute and collaborate together:
+
+* 🎯 **Focus & Smallness** — Aim each issue or change at a single, clear intention and keep work bite-sized. **Small, focused contributions** make learning and review easier.
+* 🧭 **State Intention** — Say *why* you’re doing something before you do it. Use a short user story (*“As a member…, I want…, so that…”*) in the **Issue** or **Pull Request** description to reveal the value behind the change. A clear intention guides better execution and kinder feedback.
+* 🤝 **Calm, Human Reviews** — *Assume good intent* and give **specific**, **curious**, and **modest** feedback. Reviews are for learning together, not for scoring points.
+* 🛡️ **Open Stewardship & Clear Trail** — Be transparent while respecting stewardship: welcome participation, let roles form through practice, and keep history readable (we use [***Conventional Commits***](https://www.conventionalcommits.org/en/v1.0.0/) to help with that).
+
+### ✅ How We Review
 
-- **Minimal by design** — Space is held for emergent agreements.
-- **Human-first, Agile-friendly** — We value presence over process.
-- **Conventional Commits** — Please use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format for commit messages (`feat:`, `fix:`, `docs:`, etc.)
-- **Open by invitation** — We value transparency, but also stewardship.
+* **Responds with clarity and curiosity**
+* **Offers specific suggestions rather than judgments**
+* **Remains modest and open to being changed by another’s point of view**
+* **Treats reviews as a shared opportunity for practice and growth**
+
+### 🤝 Joining the Community
+
+If you’re new here, you’re welcome to **reach out to an existing member** when starting your first contribution. There is *no formal mentorship structure* — just a space for connection and shared learning. Let the conversation grow naturally, and see what emerges.
```

Comment:
Mentorship is encouraged, highly aligned with Agile Mindset, but not formally structured

Please adjust the sentence to reflect this